jobs/kpi-forecasting/kpi_forecasting/tests/test_prophet_forecast.py

```diff
@@ -0,0 +1,458 @@
+import pandas as pd
+from dotmap import DotMap
+import numpy as np
+
+
+from kpi_forecasting.models.prophet_forecast import ProphetForecast
+
+
+def test_summarize_non_overlapping_day():
+    observed_start_date = "2124-01-01"
+    observed_end_date = "2124-02-01"
+
+    predict_start_date = "2124-02-02"
+    predict_end_date = "2124-03-01"
```

Comment:
nit -- This is simply picking dates far in the future, right? Sometimes I like to use the year 3000... "2124" is so specific that it could be confusing or misinterpreted as a typo. A comment could also help here.

Reply:
Good call, changed to 2100 (3000 raises an error, I think unix seconds get messed up sometime in the 2100s so maybe it's related to that?)